src/Database/Migrations/2022-12-13-200557_add_ban_columns.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeIgniter\Shield\Database\Migrations;
+
+use CodeIgniter\Database\Migration;
+
+class AddBanColumns extends Migration
+{
+    public function up(): void
+    {
+        // Users Table
+        $fields = [
+            'banned'      => ['type' => 'tinyint', 'after' => 'active', 'constraint' => 1, 'null' => 0, 'default' => 0],
+            'ban_message' => ['type' => 'varchar', 'after' => 'banned', 'constraint' => 255, 'null' => true],
+        ];
+
+        $this->forge->addColumn('users', $fields);
```

Comment:
Now we can customize the table names.
See https://github.com/codeigniter4/shield/blob/6197194dc3e8cfa76be470c3186d0d50938c1646/src/Database/Migrations/2020-12-28-223112_create_auth_tables.php#L43